glider_dac/views/deployment.py

```diff
@@ -364,7 +365,87 @@ def delete_deployment(username, deployment_id):
 @app.route('/api/deployment', methods=['GET'])
 @cross_origin()
 def get_deployments():
-    deployments = db.Deployment.find()
+    '''
+    API endpoint to fetch deployment info
+        ---
+    parameters:
+      - in: query
+        name: completed
+        required: false
+        schema:
+          type: string
+          enum: ['false', 'true']
+        description: >
+          Filter datasets by the completed attribute
+      - in: query
+        name: delayed_mode
+        required: false
+        schema:
+          type: string
+          enum: ['false', 'true']
+        description: >
+          Filter datasets by the delayed_mode attribute
+      - in: query
+        name: minTime
+        required: false
+        schema:
+          type: string
```

Comment:
It looks like `format` and `pattern` clash with each other, so probably just use `pattern` in that case with a regex such as the following which will support both the ISO datetimes as well as relative offsets from the current date:
`pattern: ^(?:now\s*-\s*\d+\s+hrs?|(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]+)?(Z)?)$`

Reply:
theres some simple validation below in the flask route. I'm stealing the implementation of minTime from ERDDAP as I think users are already familiar with this syntax. I believe I've tested that the ISO datetime string and 'now-<units>' works as advertised. Have you found any edge cases that arent working?

Reply:
i do like the idea of adding validation to SwaggerUI tho. Can we ticket it as an enhancement. One thing to note with your example regex, the units don't have to be hrs. It should handle any time unit that can be parsed by UDUnits.

Reply:
I fiddled around on a separate branch last night attempting to get validation to work properly.  I wasn't fully able to figure it out using Flasgger -- you might have to use `swag_from('some_definition.yml')` instead of storing the definition in a docstring.  I had issues getting the validation to apply.  We can revisit this at a later time as requested above.